windows_install_service.bat

```diff
@@ -0,0 +1,14 @@
+@echo off
+
+setlocal
+
+set SERVICE=cloud-sql-proxy
+set DISPLAYNAME=Google Cloud SQL Auth Proxy
+set CREDENTIALSFILE=%~dp0key.json
+set CONNECTIONNAME=project-id:region:db-instance
+
+sc.exe create "%SERVICE%" binPath= "\"%~dp0cloud-sql-proxy.exe\" --credentials-file \"%CREDENTIALSFILE%\" %CONNECTIONNAME%" obj= "NT AUTHORITY\Network Service" start= auto displayName= "%DISPLAYNAME%"
```

Comment:
I am not a big fan of adding comments to batch files (mainly because of the bad formatting). However, I added information to the readme file about the settings.